src/_pytest/terminal.py

```diff
@@ -1467,7 +1474,9 @@ def _get_line_with_reprcrash_message(
     except AttributeError:
         pass
     else:
-        if running_on_ci() or config.option.verbose >= 2:
+        if (
+            running_on_ci() or config.option.verbose >= 2
```

Comment:
This one is getting tricky 

But I'm torn about whether there should be a helper